js/everything.js

```diff
@@ -490,13 +510,16 @@ $(function() {
     // Handle form submission
     $("#form").submit(function(event) {
         event.preventDefault();
-
         var formValue = $.trim($("#v").val());
+        var formValueTime = /&t=(\d+)$/g.exec(formValue);
+        if (formValueTime && formValueTime.length > 1) {
+            formValueTime = parseInt(formValueTime[1], 10);
+            formValue = formValue.replace(/(&t=\d*)$/g, "");
```

Comment:
The `\d+` was a mistake. I removed the second parentheses, I needed the first one to capture the time value.